package/MDAnalysis/analysis/base.py

```diff
@@ -402,20 +706,42 @@ def run(self, start=None, stop=None, step=None, frames=None,
         step : int, optional
             number of frames to skip between each analysed frame
         frames : array_like, optional
-            array of integers or booleans to slice trajectory; `frames` can
-            only be used *instead* of `start`, `stop`, and `step`. Setting
-            *both* `frames` and at least one of `start`, `stop`, `step` to a
-            non-default value will raise a :exc:`ValueError`.
+            array of integers or booleans to slice trajectory; ``frames`` can
+            only be used *instead* of ``start``, ``stop``, and ``step``. Setting
+            *both* ``frames`` and at least one of ``start``, ``stop``, ``step``
+            to a non-default value will raise a :exc:`ValueError`.
 
             .. versionadded:: 2.2.0
-
         verbose : bool, optional
             Turn on verbosity
 
+            .. versionadded:: 2.5.0
```

Comment:
Shouldn't this be for `progressbar_kwargs` not verbose?

Reply:
it was like this:

<img width="1129" alt="image" src="https://github.com/user-attachments/assets/d330d093-9f76-4563-afcc-0d80d6683f47">

not sure what's logic for `versionadded`, but indeed this one duplicates the versionchanged below. I removed it in